rules/falco_rules.yaml

```
@@ -2065,6 +2065,7 @@
     '"sh -c openssl version"',
     '"bash -c id -Gn kafadmin"',
     '"sh -c /bin/sh -c ''date +%%s''"'
+    '"sh -c /usr/share/lighttpd/create-mime.conf.pl"'
```

```suggestion
    '"sh -c /bin/sh -c ''date +%%s''"',
    '"sh -c /usr/share/lighttpd/create-mime.conf.pl"'
```

Just a missing comma. Indeed, tests are failing for that :point_down: 
```
Could not load YAML file: yaml-cpp: error at line 2068, column 5: end of sequence flow not found
```

---

whoopsie, sorry about that!